Integrations/README.md

```diff
@@ -1,11 +0,0 @@
-### Building wheel with docker
```

Comment:
I'm concerned that we should just delete this file, rather than leaving it empty.